cmd/kots/cli/backup-create-rbac-permissions.go

```diff
@@ -0,0 +1,38 @@
+package cli
+
+import (
+	"github.com/replicatedhq/kots/pkg/snapshot"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func BackupCreateRBACPermissionsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "create-rbac-permissions",
+		Short:         "Creates the necessary minimal RBAC permissions that enables the Admin Console to access Velero.",
```

Comment:
This doesn't look velero specific.  How about
```
Creates the necessary minimal RBAC permissions for the Admin Console.
```

Reply:
It is velero specific though, this cli command will find the velero namespace and create the role and rolebinding there

Reply:
then `create-rbac-permissions` is not the right name for the command.  Also it applies all roles and binding, not just velero ones